From 1f94ae3688993f14ec5d6db8c02d999682dc4748 Mon Sep 17 00:00:00 2001
From: Saurabh Sengar <saurabh.singh@xilinx.com>
Date: Wed, 2 May 2018 13:49:42 +0530
Subject: [PATCH 0351/1851] dt-bindings: display: xlnx: mixer: Added
 xlnx,num-layers description

commit 951685be2cc8023a291126cb5af4834024c7a85e from
https://github.com/Xilinx/linux-xlnx.git

Added xlnx,num-layers property description

Signed-off-by: Saurabh Sengar <saurabh.singh@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
index a9b1907354fc..53f37e2b1a6d 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
@@ -16,6 +16,7 @@ Required properties:
  - xlnx,dma-addr-width: dma address width, valid values are 32 and 64
  - xlnx,bpc: bits per component for mixer
  - xlnx,ppc: pixel per clock for mixer
+ - xlnx,num-layers: Total number of layers (excluding logo)
  - layer_[x]: node for [x] layer
  - xlnx,layer-id: layer identifier number
  - xlnx,vformat: video format for layer. See list of supported formats below.
-- 
2.31.1

